proto/eth/v1alpha1/slasher.proto

```diff
@@ -47,3 +47,12 @@ message ProposerSlashingRequest {
 message ProposerSlashingResponse {
     repeated ethereum.eth.v1alpha1.ProposerSlashing proposer_slashing=1;
 }
+
+message MinMaxSpan {
+    uint32 min_span=1;
```

Comment:
Please add descriptions on what MinMaxSpan is

Reply:
added. thanks

Reply:
Also formatting in this file should be of the form `uint32 min_span = 1`; Maybe this proto can have better naming? `MinMaxSpan` doesn't convey much info to the reader or someone who is unfamiliar with the algorithm. I guess it should be fine it the link to the protolambda README is provided, but just thinking if there are better names...